bundles/org.openhab.binding.airq/README.md

```diff
@@ -0,0 +1,225 @@
+# air-Q Binding
+
+The air-Q Binding integrates the air analyzer <a href="http://www.air-q.com">air-Q</a> device into the openHAB system.
+
+With the binding, it is possible to subscribe to all data delivered by the air-Q device.
+
+<img src="https://uploads-ssl.webflow.com/5bd9feee2fb42232fe1d0196/5e4a8dc0e322ca33891b51e4_air-Q%20frontal-p-800.png" alt="air-Q image" width="400px" height="324px" />
+
+## Supported Things
+
+Only one Thing is supported: The `air-Q` device.
+This Binding was tested with an `air-Q Pro` device with 14 sensors. It also works with an `air-Q` device with 11 sensors.
+
+## Discovery
+
+Auto-discovery is not possible in this version.
+Since the binding has to be configured at least with the password of the device, auto-discovery would be of limited value anyway.
```

Comment:
Suggest removing this sentence and just simply state "Auto-discovery is not yet supported".